grpclib/client.py

```diff
@@ -68,11 +68,12 @@ class Stream(StreamIterator):
     initial_metadata = None
     trailing_metadata = None
 
-    def __init__(self, channel, request, send_type, recv_type):
+    def __init__(self, channel, request, send_type, recv_type, *, codec=None):
```

Comment:
I think that there is no need to make this `codec` argument optional, which have some default value. It would be better if only Channel class is responsible for this.